test/refunder.ts

```diff
@@ -58,9 +41,16 @@ describe("Refunder", function() {
 		greeter = await Greeter.deploy(greeterInitGreet);
 		await greeter.deployed();
 
+		const Registry = await ethers.getContractFactory("Registry");
+		registry = await Registry.deploy();
+		await registry.deployed();
+
 		const Refunder = await ethers.getContractFactory("Refunder");
 		refunder = await Refunder.deploy();
 		await refunder.deployed();
+		await refunder.init(owner.address);
+
+		await registry.register(refunder.address, REFUNDER_VERSION);
```

Comment:
Shouldn't we use the factory?

Reply:
as you said, the user should be able to register their refunders without factory, and in time of writing the refunder tests there was no Factory :)